### ISSUE TITLE
[Cosmetic bug] Duplicate "-nographic" option

### ISSUE THREAD

**Issue by adosztal** [GNS3/gns3-gui#2265]
This is the command line used to start qemu nodes:
`/usr/bin/qemu-system-x86_64 [...] -nographic -nographic`

The option appears only once at the "Additional settings":
![nographic](https://user-images.githubusercontent.com/16632791/30310691-018468cc-9792-11e7-9ddd-853f8c2b138a.png)

The template does not contain "-nographic" at all:
```json
    "qemu": {
        "adapter_type": "e1000",
        "adapters": 3,
        "ram": 512,
        "arch": "x86_64",
        "console_type": "telnet",
        "boot_priority": "dc",
        "kvm": "allow"
    },
```

Seems like gns3-server adds this option anyway when dragging in a node, even if it's already there.

I suggest to remove this feature from the import procedure. I think [this is the appropriate section](https://github.com/GNS3/gns3-gui/blob/3d21f9a997e471ff295c66e209d569b13775efc8/gns3/registry/config.py#L222) but I'd need a confirmation before submitting a PR. :)

**Comment by grossmj** [comment]
We force that option because Qemu would no start on some platforms (Windows and/or OSX) if I remember correctly. Also, I am afraid of side effects if we remove this just before we release 2.1

**Comment by adosztal** [comment]
I know it should be there, it just shouldn't appear in the command line twice. Anyway, it's really just a cosmetic issue that can be indeed postponed to 2.2.

**Comment by grossmj** [comment]
Fixed in https://github.com/GNS3/gns3-server/commit/cfbcc1194d07c032ce912720fd97c159d2db5c59
